### ISSUE TITLE
[ios] can't run template

### ISSUE THREAD

**Issue by pauliusguzas** [flexn-io/renative#1128]
**Describe the bug**
Can't run template on ios

**To Reproduce**
1. go to template-starter package
2. run `rnv run -p ios`

```
┌──────────────────────────────────────────────────────────────────────────────┐
│  🚀  BUNDLER STARTED | 10/5/2023, 3:23:07 PM                                 │
│  $ rnv run -p ios                                                            │
├──────────────────────────────────────────────────────────────────────────────┤
│  ReNative Version: 1.0.0-canary.1                                            │
│  Project Name ($package.name): @rnv/template-starter                         │
│  Project Version ($package.version): 1.0.0-canary.1                          │
│  Workspace ($.workspaceID): rnv                                              │
│  Platform (-p): ios                                                          │
│  Engine ($.platforms.ios.engine): engine-rn                                  │
│  App Config (-c): template                                                   │
│  Build Scheme (-s): debug                                                    │
│  Target (-t): iPhone 14                                                      │
│  Supported Platforms: ios                                                    │
│  Env Info: darwin | x64 | node v18.16.0                                      │
│  Executed Time: 0h:0m:45s:986ms                                              │
│                                                                              │
│ [ warn ] [project configure] Version mismatch of devDependency react-native-gesture-handler between package.json: v(2.13.1) and plugins.json: v(2.12.1). This is template. NO ACTION
│                                                                              │
│  Project location:                                                           │
│  ./platformBuilds/template_ios/RNVApp.xcworkspace                            │
└──────────────────────────────────────────────────────────────────────────────┘
Error: Duplicated files or mocks. Please check the console for more info
    at setModule (/Users/pauliusguzas/Desktop/renative/renative/node_modules/metro-file-map/src/index.js:450:17)
    at workerReply (/Users/pauliusguzas/Desktop/renative/renative/node_modules/metro-file-map/src/index.js:507:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Promise.all (index 2450)
    at HasteMap._applyFileDelta (/Users/pauliusguzas/Desktop/renative/renative/node_modules/metro-file-map/src/index.js:672:7)
    at /Users/pauliusguzas/Desktop/renative/renative/node_modules/metro-file-map/src/index.js:327:9
    at DependencyGraph.ready (/Users/pauliusguzas/Desktop/renative/renative/node_modules/metro/src/node-haste/DependencyGraph.js:95:5)
    at Bundler.ready (/Users/pauliusguzas/Desktop/renative/renative/node_modules/metro/src/Bundler.js:69:5)
    at IncrementalBundler.ready (/Users/pauliusguzas/Desktop/renative/renative/node_modules/metro/src/IncrementalBundler.js:289:5)
    at Server.ready (/Users/pauliusguzas/Desktop/renative/renative/node_modules/metro/src/Server.js:1085:5)
```
<img width="1102" alt="Screenshot 2023-10-05 at 15 25 29" src="https://github.com/flexn-io/renative/assets/65670660/ea9a513b-35f3-46e5-91e7-3a1c2b0996ac">

**Desktop (please complete the following information):**

-   OS: Ventura 13.5.2
-   Node Version 18.16.0
-   RNV Version 1.0.0-canary.1
-   rnv/template-starter Version 1.0.0-canary.1

**Comment by ElenaDiachenko** [comment]
It works for me

**Comment by pauliusguzas** [comment]
checked out from sanity_1.0.0-1 tag
```
 $ rnv run -p ios - Done! 🚀
┌──────────────────────────────────────────────────────────────────────────────┐
│  🚀  SUMMARY | 10/6/2023, 1:51:12 PM                                         │
│  $ rnv run -p ios                                                            │
├──────────────────────────────────────────────────────────────────────────────┤
│  ReNative Version: 1.0.0-canary.1                                            │
│  Project Name ($package.name): @rnv/template-starter                         │
│  Project Version ($package.version): 1.0.0-canary.1                          │
│  Workspace ($.workspaceID): rnv                                              │
│  Platform (-p): ios                                                          │
│  Engine ($.platforms.ios.engine): engine-rn                                  │
│  App Config (-c): template                                                   │
│  Build Scheme (-s): debug                                                    │
│  Target (-t): iPhone 14                                                      │
│  Supported Platforms: ios                                                    │
│  Env Info: darwin | x64 | node v18.16.0                                      │
│  Executed Time: 0h:0m:5s:416ms                                               │
│                                                                              │
│ [ warn ] [configure] pod install is not enough! Let's try pod update! Error:
 COMMAND:

bundle exec pod install

FAILED with ERROR:

Calling `DidYouMean::SPELL_CHECKERS.merge!(error_name => spell_checker)' has been deprecated. Please call `DidYouMean.correct_error(error_name, spell_checker)' instead.
bundler: failed to load command: pod (./vendor/bundle/ruby/3.1.0/bin/pod)
./vendor/bundle/ruby/3.1.0/gems/activesupport-7.1.0/lib/active_support/core_ext/array/conversions.rb:108:in `<class:Array>': undefined method `deprecator' for ActiveSupport:Module (NoMethodError)
 deprecate to_default_s: :to_s, deprecator: ActiveSupport.deprecator
 ^^^^^^^^^^^
Did you mean? deprecate_constant
	from ./vendor/bundle/ruby/3.1.0/gems/activesupport-7.1.0/lib/active_support/core_ext/array/conversions.rb:8:in `<top (required)>'
	from ./vendor/bundle/ruby/3.1.0/gems/cocoapods-1.13.0/lib/cocoapods.rb:9:in `require'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/lib/bundler/friendly_errors.rb:128:in `with_friendly_errors'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/exe/bundle:37:in `<top (required)>'
	from /Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/bin/bundle:25:in `load'
	from /Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/bin/bundle:25:in `<main>'
	from /Users/pauliusguzas/.rvm/gems/ruby-3.1.1/bin/ruby_executable_hooks:22:in `eval'
	from /Users/pauliusguzas/.rvm/gems/ruby-3.1.1/bin/ruby_executable_hooks:22:in `<main>'

Command failed with exit code 1: bundle exec pod install
Calling `DidYouMean::SPELL_CHECKERS.merge!(error_name => spell_checker)' has been deprecated. Please call `DidYouMean.correct_error(error_name, spell_checker)' instead.
bundler: failed to load command: pod (./vendor/bundle/ruby/3.1.0/bin/pod)
./vendor/bundle/ruby/3.1.0/gems/activesupport-7.1.0/lib/active_support/core_ext/array/conversions.rb:108:in `<class:Array>': undefined method `deprecator' for ActiveSupport:Module (NoMethodError)

  deprecate to_default_s: :to_s, deprecator: ActiveSupport.deprecator
                                                          ^^^^^^^^^^^
Did you mean?  deprecate_constant
	from ./vendor/bundle/ruby/3.1.0/gems/activesupport-7.1.0/lib/active_support/core_ext/array/conversions.rb:8:in `<top (required)>'
	from ./vendor/bundle/ruby/3.1.0/gems/cocoapods-1.13.0/lib/cocoapods.rb:9:in `require'
	from ./vendor/bundle/ruby/3.1.0/gems/cocoapods-1.13.0/lib/cocoapods.rb:9:in `<top (required)>'
	from ./vendor/bundle/ruby/3.1.0/gems/cocoapods-1.13.0/bin/pod:36:in `require'
	from ./vendor/bundle/ruby/3.1.0/gems/cocoapods-1.13.0/bin/pod:36:in `<top (required)>'
	from ./vendor/bundle/ruby/3.1.0/bin/pod:25:in `load'
	from ./vendor/bundle/ruby/3.1.0/bin/pod:25:in `<top (required)>'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/lib/bundler/cli/exec.rb:63:in `load'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/lib/bundler/cli/exec.rb:63:in `kernel_load'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/lib/bundler/cli/exec.rb:28:in `run'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/lib/bundler/cli.rb:474:in `exec'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/lib/bundler/vendor/thor/lib/thor.rb:392:in `dispatch'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/lib/bundler/cli.rb:30:in `dispatch'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/lib/bundler/vendor/thor/lib/thor/base.rb:485:in `start'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/lib/bundler/cli.rb:24:in `start'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/exe/bundle:49:in `block in <top (required)>'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/lib/bundler/friendly_errors.rb:128:in `with_friendly_errors'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/exe/bundle:37:in `<top (required)>'
	from /Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/bin/bundle:25:in `load'
	from /Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/bin/bundle:25:in `<main>'
	from /Users/pauliusguzas/.rvm/gems/ruby-3.1.1/bin/ruby_executable_hooks:22:in `eval'
	from /Users/pauliusguzas/.rvm/gems/ruby-3.1.1/bin/ruby_executable_hooks:22:in `<main>'

Calling `DidYouMean::SPELL_CHECKERS.merge!(error_name => spell_checker)' has been deprecated. Please call `DidYouMean.correct_error(error_name, spell_checker)' instead.
bundler: failed to load command: pod (./vendor/bundle/ruby/3.1.0/bin/pod)
./vendor/bundle/ruby/3.1.0/gems/activesupport-7.1.0/lib/active_support/core_ext/array/conversions.rb:108:in `<class:Array>': undefined method `deprecator' for ActiveSupport:Module (NoMethodError)

  deprecate to_default_s: :to_s, deprecator: ActiveSupport.deprecator
                                                          ^^^^^^^^^^^
Did you mean?  deprecate_constant
	from ./vendor/bundle/ruby/3.1.0/gems/activesupport-7.1.0/lib/active_support/core_ext/array/conversions.rb:8:in `<top (required)>'
	from ./vendor/bundle/ruby/3.1.0/gems/cocoapods-1.13.0/lib/cocoapods.rb:9:in `require'
	from ./vendor/bundle/ruby/3.1.0/gems/cocoapods-1.13.0/lib/cocoapods.rb:9:in `<top (required)>'
	from ./vendor/bundle/ruby/3.1.0/gems/cocoapods-1.13.0/bin/pod:36:in `require'
	from ./vendor/bundle/ruby/3.1.0/gems/cocoapods-1.13.0/bin/pod:36:in `<top (required)>'
	from ./vendor/bundle/ruby/3.1.0/bin/pod:25:in `load'
	from ./vendor/bundle/ruby/3.1.0/bin/pod:25:in `<top (required)>'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/lib/bundler/cli/exec.rb:63:in `load'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/lib/bundler/cli/exec.rb:63:in `kernel_load'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/lib/bundler/cli/exec.rb:28:in `run'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/lib/bundler/cli.rb:474:in `exec'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/lib/bundler/vendor/thor/lib/thor.rb:392:in `dispatch'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/lib/bundler/cli.rb:30:in `dispatch'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/lib/bundler/vendor/thor/lib/thor/base.rb:485:in `start'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/lib/bundler/cli.rb:24:in `start'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/exe/bundle:49:in `block in <top (required)>'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/lib/bundler/friendly_errors.rb:128:in `with_friendly_errors'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/exe/bundle:37:in `<top (required)>'
	from /Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/bin/bundle:25:in `load'
	from /Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/bin/bundle:25:in `<main>'
	from /Users/pauliusguzas/.rvm/gems/ruby-3.1.1/bin/ruby_executable_hooks:22:in `eval'
	from /Users/pauliusguzas/.rvm/gems/ruby-3.1.1/bin/ruby_executable_hooks:22:in `<main>'

Error: Command failed with exit code 1: bundle exec pod install
Calling `DidYouMean::SPELL_CHECKERS.merge!(error_name => spell_checker)' has been deprecated. Please call `DidYouMean.correct_error(error_name, spell_checker)' instead.
bundler: failed to load command: pod (./vendor/bundle/ruby/3.1.0/bin/pod)
./vendor/bundle/ruby/3.1.0/gems/activesupport-7.1.0/lib/active_support/core_ext/array/conversions.rb:108:in `<class:Array>': undefined method `deprecator' for ActiveSupport:Module (NoMethodError)

  deprecate to_default_s: :to_s, deprecator: ActiveSupport.deprecator
                                                          ^^^^^^^^^^^
Did you mean?  deprecate_constant
	from ./vendor/bundle/ruby/3.1.0/gems/activesupport-7.1.0/lib/active_support/core_ext/array/conversions.rb:8:in `<top (required)>'
	from ./vendor/bundle/ruby/3.1.0/gems/cocoapods-1.13.0/lib/cocoapods.rb:9:in `require'
	from ./vendor/bundle/ruby/3.1.0/gems/cocoapods-1.13.0/lib/cocoapods.rb:9:in `<top (required)>'
	from ./vendor/bundle/ruby/3.1.0/gems/cocoapods-1.13.0/bin/pod:36:in `require'
	from ./vendor/bundle/ruby/3.1.0/gems/cocoapods-1.13.0/bin/pod:36:in `<top (required)>'
	from ./vendor/bundle/ruby/3.1.0/bin/pod:25:in `load'
	from ./vendor/bundle/ruby/3.1.0/bin/pod:25:in `<top (required)>'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/lib/bundler/cli/exec.rb:63:in `load'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/lib/bundler/cli/exec.rb:63:in `kernel_load'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/lib/bundler/cli/exec.rb:28:in `run'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/lib/bundler/cli.rb:474:in `exec'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/lib/bundler/vendor/thor/lib/thor.rb:392:in `dispatch'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/lib/bundler/cli.rb:30:in `dispatch'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/lib/bundler/vendor/thor/lib/thor/base.rb:485:in `start'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/lib/bundler/cli.rb:24:in `start'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/exe/bundle:49:in `block in <top (required)>'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/lib/bundler/friendly_errors.rb:128:in `with_friendly_errors'
	from ./vendor/bundle/ruby/3.1.0/gems/bundler-2.2.22/exe/bundle:37:in `<top (required)>'
	from /Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/bin/bundle:25:in `load'
	from /Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/bin/bundle:25:in `<main>'
	from /Users/pauliusguzas/.rvm/gems/ruby-3.1.1/bin/ruby_executable_hooks:22:in `eval'
	from /Users/pauliusguzas/.rvm/gems/ruby-3.1.1/bin/ruby_executable_hooks:22:in `<main>'
    at makeError (/Users/pauliusguzas/Desktop/renative/renative/node_modules/execa/lib/error.js:60:11)
    at handlePromise (/Users/pauliusguzas/Desktop/renative/renative/node_modules/execa/index.js:118:26)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)


│                                                                              │
│  Project location:                                                           │
│  ./platformBuilds/template_ios/RNVApp.xcworkspace                            │
└──────────────────────────────────────────────────────────────────────────────┘
pauliusguzas@MBP-de-Flexn template-starter %
```

**Comment by ElenaDiachenko** [comment]
fixed in 1.0.0-canary.7.
